packages/mdc-toolbar/mdc-toolbar.scss

```diff
@@ -88,6 +91,94 @@ $mdc-toolbar-mobile-breakpoint: 599px;
   @include mdc-elevation(4);
 }
 
+.mdc-toolbar--fixed-lastrow-only {
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 1;
+
+  @include mdc-elevation(4);
+}
+
+.mdc-toolbar--flexible {
+  // stylelint-disable plugin/selector-bem-pattern
+  // bem linter is disabled because it cannot correctly reconcile mdc prefix for css variables.
+  --mdc-toolbar-ratio-to-extend-flexible: $mdc-toolbar-ratio-to-extend-flexible;
+  // stylelint-enable plugin/selector-bem-pattern
+
+  .mdc-toolbar__row:first-child {
+    height: $mdc-toolbar-row-height * $mdc-toolbar-ratio-to-extend-flexible;
+    height: calc(#{$mdc-toolbar-row-height} * var(--mdc-toolbar-ratio-to-extend-flexible, #{$mdc-toolbar-ratio-to-extend-flexible}));
+
+    @media (max-width: $mdc-toolbar-mobile-breakpoint) {
+      height: $mdc-toolbar-mobile-row-height * $mdc-toolbar-ratio-to-extend-flexible;
+      height: calc(#{$mdc-toolbar-mobile-row-height} * var(--mdc-toolbar-ratio-to-extend-flexible, #{$mdc-toolbar-ratio-to-extend-flexible}));
+    }
+
+    &::after {
```

Comment:
ack, SG. I'd just add a comment making this explicit, e.g. "ensure that `::after` will be defined so it can be easily styled"